lib/services/lwm2mHandlers/commonLwm2m.js

```diff
@@ -81,8 +81,8 @@ function observeActiveAttributes(payload, registeredDevice, callback) {
         activeAttributes = [],
```

Comment:
CNR entry missing. A possiblity could be:

```
- Fix: observer creation fails for preregistered devices (#107)
```

Observer, is a very developer-oriented wording. If could find an alternative from IOTA user perspetive (which impact has the bug in usage?) would be better.

Reply:
Fixed in 379ac92eecda596c498556fd27a7316396ec7510